tests/drivers/display/display_read_write/src/main.c

```diff
@@ -26,7 +32,11 @@ static inline uint8_t bytes_per_pixel(enum display_pixel_format pixel_format)
 	case PIXEL_FORMAT_ARGB_8888:
 		return 4;
 	case PIXEL_FORMAT_RGB_888:
+#if defined(CONFIG_RENESAS_RA_GLCDC)
```

Comment:
We cannot do this. Weren't we able to report the `ARGB_8888` pixel format from the driver to resolve this same issue in the LVGL glue code?

Reply:
I revert this change